packages/studio-plugin/tests/__utils__/createTestSourceFile.ts

```diff
@@ -1,13 +1,21 @@
-import { createTsMorphProject } from "../../src/ParsingOrchestrator";
+import { Project } from "ts-morph";
 
 export default function createTestSourceFile(
```

Comment:
Looking at this method again, it's a bit confusing. It does more than simply create a SourceFile, it also creates a new Project. If we were to have this method take in a `ts-morph` project, would that speed up our unit tests at all? Or not really since when you add a SourceFile it recreates the TypeScript Compiler Project anyway. My thought was in a test file, such as `TypeNodeParsingHelper.test.ts`, we could have 
```
const globalProject = createTestProject(); 
```
at the top of the file. Then, in the individual tests have `createTestSourceFile([whatever Code], globalProject, [optional filePath[)`.

Reply:
I think it would speed up unit tests that share the same source files, but unit tests that call `createTestSourceFile` call it to test some individual bit of code, so I doubt there would be performance benefit from sharing the Project instance